polaris-react/scripts/readme-update-version.js

```diff
@@ -9,7 +9,7 @@ const {semverRegExp} = require('../scripts/utilities');
 
 const root = resolve(__dirname, '..');
 
-const readmes = Object.freeze(['README.md', 'src/components/README.md']);
+const readmes = Object.freeze(['README.md']);
```

Comment:
Now we're down to one item, and I can't see us adding additional files in the future, this we don't really need this to loop. Can be tidied in a follow-up.